model/riscv_platform.sail

```diff
@@ -210,22 +210,14 @@ function clint_load(t, addr, width) = {
 }
 
 function clint_dispatch() -> unit = {
-  if   get_config_print_platform()
-  then print_platform("clint::tick mtime <- " ^ BitStr(mtime));
-  mip[MTI] = 0b0;
-  if mtimecmp <=_u mtime then {
-    if   get_config_print_platform()
-    then print_platform(" clint timer pending at mtime " ^ BitStr(mtime));
-    mip[MTI] = 0b1;
-  };
+  mip[MTI] = bool_to_bits(mtimecmp <=_u mtime);
+  if get_config_print_platform()
+  then print_platform(" clint mtime " ^ BitStr(mtime) ^ " (mip.MTI <- " ^ BitStr(mip[MTI]) ^ ")");
   /* Sstc - supervisor timer register */
   if extensionEnabled(Ext_Sstc) then {
-    mip[STI] = 0b0;
-    if stimecmp <=_u mtime then {
-      if   get_config_print_platform()
-      then print_platform(" supervisor timer pending at mtime " ^ BitStr(mtime));
-      mip[STI] = 0b1;
-    }
+    mip[STI] = bool_to_bits(stimecmp <=_u mtime); 
+    if get_config_print_platform()
+    then print_platform(" clint mtime " ^ BitStr(mtime) ^ " (mip.STI <- " ^ BitStr(mip[STI]) ^ ")");
```

Comment:
Why do these have a leading space? And why is this the same string as above?

Reply:
It's not exactly the same - it prints STI instead of MTI.

But I think having two prints is needlessly verbose. @Ali-Faraz-10xe can you change it to this?

```
function clint_dispatch() -> unit = {
  mip[MTI] = bool_to_bits(mtimecmp <=_u mtime);
  if extensionEnabled(Ext_Sstc) then {
    mip[STI] = bool_to_bits(stimecmp <=_u mtime); 
  }
  
  if get_config_print_platform()
  then print_platform(" clint mtime " ^ BitStr(mtime) ^ " (mip.MTI <- " ^ BitStr(mip[MTI]) ^
    (if extensionEnabled(Ext_Sstc) then ", mip.STI <- " ^ BitStr(mip[STI]) else "") ^ ")");
}
```

Reply:
I still don't understand the desire to have the leading space though

Reply:
Ah yeah I think that made sense with the output before when it was a separate line after the `clint::tick` one, but we can remove it now.